app/src/main/java/com/kickstarter/viewmodels/AccountViewModel.kt

```diff
@@ -114,7 +114,7 @@ interface AccountViewModel {
         private fun showEmailErrorImage(userPrivacy: UserPrivacyQuery.Data?): Boolean? {
             return if (userPrivacy?.me()?.isCreator!! && !userPrivacy.me()?.isDeliverable!! || !userPrivacy.me()?.isEmailVerified!!) {
```

Comment:
you're going to be sad but i realized the logic is simpler like this:
 ```private fun showEmailErrorImage(userPrivacy: UserPrivacyQuery.Data?): Boolean? {
            return if ( !userPrivacy.me()?.isDeliverable!!) {
                return true
            } else if (userPrivacy?.me()?.isCreator!! && !userPrivacy.me()?.isEmailVerified!!) {
                return true
            } else {
                false
            }
        }
```

Reply:
fixed!